scripts/core/kernel.yml

```diff
@@ -190,6 +190,18 @@ params:
       name: "pPropSizeRet"
       desc: |
             [out][optional] pointer to the actual size in bytes of data being queried by propName.
+returns:
+    - $X_RESULT_ERROR_UNSUPPORTED_ENUMERATION:
+        - "If `propName` is not supported by the adapter."
```

Comment:
this line seems one indentation too far in (here and a few other files)

Reply:
Tabs are 4 spaces in the project, its all the other ones that don't have the correct intendation.

Reply:
Indented to 4-spaces 👍